graphql/resolvers.test.js

```diff
@@ -1,7 +1,7 @@
-jest.mock('node-fetch')
-jest.mock('mailgun-js')
 import resolvers from '../graphql/resolvers'
 import db from '../helpers/dbload'
+jest.mock('node-fetch')
+jest.mock('mailgun-js')
 
```

Comment:
`jest.mock('../helpers/dbload')`

This will auto-mock every import from this folder, save a few lines :)

Reply:
Just remember to use `jest.resetAllMocks` in `beforeEach` that way specific mocks don't carry over to other tests

Reply:
sorry, this looks like the wrong file for this lol
and i think you need to add back the 
```
jest.mock('node-fetch')	
jest.mock('mailgun-js')
```

Reply:
the above comments are for the tests that you created for this PR